### PR TITLE
added minikube docker-env docs

### PR DESCRIPTION
**Pull request by jstrachan**
thanks to @aaron-prindle for the heads up!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,20 @@ Stopping local Kubernetes cluster...
 Stopping "minikubeVM"...
 ```
 
+### Reusing the Docker daemon
+
+When using a single VM of kubernetes its really handy to reuse the Docker daemon inside the VM; as this means you don't have to build on your host machine and push the image into a docker registry - you can just build inside the same docker daemon which speeds up local experiements.
```
Comment (dlorenc) [inline README.md]: typo: experiments

File: README.md [hunk 2]
```diff
@@ -54,6 +54,20 @@ Stopping local Kubernetes cluster...
...
+
+To be able to work with the docker daemon on your mac/windows host type the following into your shell:
```
Comment (dlorenc) [inline README.md]: We don't currently support windows, remove for now? We support linux so you could swap it to that.

**Comment by dlorenc** [comment]
Thanks for the update. Two small nits, then LGTM.

**Comment by jstrachan** [comment]
@dlorenc thanks, nits squashed; added a link too to the current docs for docker-env: https://github.com/kubernetes/minikube/blob/master/docs/minikube_docker-env.md

**Comment by dlorenc** [comment]
Thanks!

**Pull request merged**